packages/blocks/table/src/Table.tsx

```diff
@@ -340,15 +346,6 @@ export const Table: BlockComponent<AppProps> = ({
           entityTypeVersionId,
         },
       ]);
-
-      void updateLinks([
```

Comment:
How come this isn't necessary anymore?

Reply:
This change was made by me during the initial refactor. However, this isn't really doing any transformation on the existing data, just re-POSTing it.

So, removing this to reduce redundancy and repeated calls to the API.